Source/Pages/Gallery/YPAssetZoomableView.swift

```diff
@@ -305,7 +302,50 @@ fileprivate extension YPAssetZoomableView {
             self?.centerAssetView()
         }
     }
-    
+
+    func findClosestAllowedAspectRatio(for aspectRatio: CGFloat) -> CGFloat {
```

Comment:
This would be nice to have a test for but we don't have that established in the image picker library nor do we have the CI infrastructure setup to run tests even if we did have them here 🤔

Reply:
Yeah true. Maybe it would be worth it to just add a simple unit test file for this, even if it's not automated in the CI.